core/src/main/java/org/elasticsearch/search/aggregations/metrics/max/InternalMax.java

```diff
@@ -90,6 +95,31 @@ public XContentBuilder doXContentBody(XContentBuilder builder, Params params) th
         return builder;
     }
 
+    private static final ObjectParser<Map<String, Object>, Void> PARSER = new ObjectParser<>(
+            "internal_max", HashMap::new);
+
+    static {
+        declareCommonField(PARSER);
+        PARSER.declareField((map, value) -> map.put(CommonFields.VALUE.getPreferredName(), value),
+                (p, c) -> InternalMax.parseValue(p, Double.NEGATIVE_INFINITY), CommonFields.VALUE,
```

Comment:
I chose Double.NEGATIVE_INFINITY because thats the default value that is used in #doReduce() as the default value.